### DISCUSSION TITLE
Profile nuked after Windows update

### DISCUSSION THREAD

**Discussion by theFuzzyWarble** [beakerbrowser/beaker#1841] [General]
After dabbling with Beaker on my linux install, I decided to created a new profile on my windows 10 machine and see what fun could be had..
I threw up some quick page code and added the profiles to each other then called it a night, saved and tested both profiles to see if messages posted to each.

Next morning, after a Windows update, I jump into Beaker[Win10] and it's prompting me to create a profile again.. :-1: 

I'll assume a fluke worst case, but I'm just putting this out there just in case it's a thing.

**Comment by pfrazee** [comment]
Oof, boo. Sorry to hear that. I have honestly no clue why that would happen. Windows shouldn't be deleting app data for updates.

**Reply by theFuzzyWarble** [reply]
No worries, 1 step back 10 forward since! Thankfully, there was nothing lost but a few minutes of code.
Not a hitch since in Windows!!
But I'll let you know NEXT update  heh.
